r/R/dplyr.R

```diff
@@ -219,6 +219,31 @@ tail.arrow_dplyr_query <- function(x, n = 6L, ...) {
   x
 }
 
+#' Show the details of an Arrow ExecPlan
+#'
+#' This is a function which gives more details about the `ExecPlan` of an
+#' `arrow_dplyr_query` object. It is similar to `dplyr::show_query()`.
+#'
+#' @param x an `arrow_dplyr_query` to print the ExecPlan for.
+#'
+#' @return The argument, invisibly.
+#' @export
+#'
+#' @examples
```

Comment:
We probably want to mark this as: 

```suggestion
#' @examplesIf arrow_with_dataset() & requireNamespace("dplyr", quietly = TRUE)
```

But definitely the {dplyr} bit, since we don't depend on {dplyr} (only suggests) and CRAN will be upset if the example fails if {dplyr} isn't available.

Reply:
Thanks! Yep, there definitely were some things I wanted to finesse. Done.